script/check-posix-pty

```diff
@@ -158,14 +158,13 @@ EOF
 
     LEADER_ALOC=fdde:ad00:beef::ff:fe00:fc00
     # Retrievie extended address through network diagnostic get
-    coap_response=$(echo -n '120100' | xxd -r -p | coap-client -m POST coap://[${LEADER_ALOC}]:61631/d/dg -f- | xxd -p | grep 0008)
-    echo "CoAP response is: ${coap_response}"
+    coap_response=$(echo -n '120100' | xxd -r -p | coap-client -B 5 -m POST coap://[${LEADER_ALOC}]:61631/d/dg -f-)
```

Comment:
FYI: 61631 won't be accessible from Host due to https://github.com/openthread/openthread/pull/5910.

I am currently trying to fix this part of test in https://github.com/openthread/openthread/pull/5910

Any suggestion on how to fix it is welcomed.